Makefile

```diff
@@ -186,8 +186,8 @@ valgrind:
 #----------------------------------------------------------------------------------------------
 
 flow_test:
-	$(SHOW)$(MAKE) pybind
-	$(SHOW)tox -e flowenv
+	$(SHOW)poetry install
```

Comment:
no need to build?

Reply:
No, poetry install will trigger a ~build locally. You only need a _poetry build_ if you want to package for staging to PYPI, etc.